src/templates/profile.ts

```diff
@@ -15,3 +16,61 @@ ${name} usecase
 usecase ${name} {}
 `;
 }
+
+export function complete(name: string): string {
+  return `
+"""
+${name} usecase
+"""
+usecase ${name} {
+  input {
+    "field title"
+    // foo is required nullable string
+    foo! string 
+    // bar links to named field 
+    bar 
+  }
+  result {
+    // optional nullable string
+    baz string
+  }
+
+  // error links to named model
+  error error
+
+  example success {
+    input {
+      foo = "example"
+      bar = true
+    }
+    result {
+      baz = "result"
+    }
+  }
+
+  example fail {
+    input {
+      foo = "error"
+      bar = false
+    }
+    error {
+      status = 400
+      title = "Not Found"
+    }
+  }
+}
+// optional nullable boolean or number
+field bar boolean | number 
+
+model error {
+  status enum {
+    NOT_FOUND = 404,
+    BAD_REQUEST = 400
+  }
+  // required string
```

Comment:
Empty space for readability
```suggestion

  // required string
```